sphinx/domains/python.py

```diff
@@ -220,6 +224,8 @@ def handle_signature(self, sig, signode):
             if retann:
                 signode += addnodes.desc_returns(retann, retann)
             if anno:
+                if anno[0] == '=':
```

Comment:
I'm not sure this is the safest approach.  This would also remove memory addresses where the user explicitly typed this out (instead of just for autodoc.)  Could we make sure this only covers autodoc'd contents?